pkg/apis/events/v1alpha1/eventsource_types.go

```diff
@@ -858,7 +858,7 @@ type GitlabEventSource struct {
 	// Metadata holds the user defined metadata which will passed along the event payload.
 	// +optional
 	Metadata map[string]string `json:"metadata,omitempty" protobuf:"bytes,9,rep,name=metadata"`
-	// List of project IDs or project namespace paths like "whynowy/test". Projects and groups cannot be empty at the same time.
+	// List of project IDs or project namespace paths like "whynowy/test".
```

Comment:
Can you add that not defining a group or project means the event source will not manage webhooks?